packages/core/src/components/AttentionBox/AttentionBox.tsx

```diff
@@ -71,7 +73,9 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
 
   return (
     <aside
-      className={cx(styles.attentionBox, getStyle(styles, camelCase(`type-${type}`)), overrideClassName)}
+      className={cx(styles.attentionBox, getStyle(styles, camelCase(`type-${type}`)), overrideClassName, {
+        [styles.animateOnEnter]: animateOnEnter
```

Comment:
should this class be removed after the animation ends? or does the animation just works once and you don't care about it afterwards?

Reply:
ok nm, I don't think it matters